src/main/java/org/jmhsrobotics/frc2023/subsystems/Drivetrain.java

```diff
@@ -248,6 +261,33 @@ public void resetOdometry(Pose2d pose) {
 		);
 		// spotless:on
 	}
+	public SequentialCommandGroup createRamseteCommand(Trajectory path) {
+		final DifferentialDriveKinematics driveKinematics = new DifferentialDriveKinematics(
+				AutoConstants.trackWidthMeters);
+
+		return new InstantCommand(() -> {
+			this.isInRamsete = true;
+		}).andThen(new RamseteCommand(path, this::getPose, new RamseteController(),
+				new SimpleMotorFeedforward(AutoConstants.trajectoryFeedForwardVolts,
+						AutoConstants.trajectoryFeedForwardVoltSecondsPerMeter,
+						AutoConstants.trajectoryFeedForwardVoltSecondsSquaredPerMeter),
```

Comment:
How these constants are used may want to be cleaned up. The constants for the drivetrain/trackwidth are not auto specific. And Likely should be defined inside of the drive ports Interface as these values are very likely to change from bot to bot.

PIDcontroler coefficients should use the PIDConfig Classes to limit number of unique constants stored in the constants file or drive ports. The PIDConfig class also includes a factory method for easly constructing PIDControllers from the configs.